source/faq.txt

```diff
@@ -75,4 +75,23 @@ The following example creates indexes on the ``movies`` collection by using
 
 To learn more about creating indexes by using the driver, see the
 `Indexes guide <{+driver-docs-root+}/fundamentals/indexes/>`__ in the
-{+csharp-driver-long+} documentation.
\ No newline at end of file
+{+csharp-driver-long+} documentation.
+
+Can I Access {+csharp-driver-short+} Features in the {+provider-short+}?
+------------------------------------------------------------------------
+
+You can use the {+csharp-driver-short+} directly in your {+provider-short+}
+application by calling the driver methods on the ``MongoClient`` used to set up
+your ``DbContext``. The {+provider-short+} is built on top of the
+{+csharp-driver-short+}, so you can use most driver features in your
+application. For example, you can use the driver to perform an Atlas Search
```

Comment:
S: since you say "most", can you provide an example of a limitation as well?

Reply:
Thanks for the callout, I checked in on this and found out that there aren't any limitations, so updated to say "all"